client/src/index.css

```diff
@@ -1,13 +1,53 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans',
+    'Droid Sans', 'Helvetica Neue', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
+}
+
+/* typing loaser */
+.typing-loader {
+  display: inline-block;
+  position: relative;
+  height: 2rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+  padding: 0 0.5rem;
+}
+.typing-loader div {
+  display: inline-block;
+  width: 0.6rem;
+  height: 0.6rem;
+  border-radius: 50%;
+  background: #ddd;
+  margin: 0 0.3rem;
+  animation: typing-loader-animation 1.2s cubic-bezier(0, 0.5, 0.5, 1) infinite;
+  position: relative;
+}
+.typing-loader div:nth-child(1) {
+  animation-delay: -0.24s;
+}
+.typing-loader div:nth-child(2) {
+  animation-delay: -0.12s;
+}
+.typing-loader div:nth-child(3) {
+  animation-delay: -0;
+}
+@keyframes typing-loader-animation {
+  0% {
+    top: 0rem;
+  }
+  25% {
+    top: 0.5rem;
+  }
+  100% {
+    top: -0.5rem;
+  }
 }
```

Comment:
is there any reason the style for a component is defined in global CSS file? can you put the style in `otherUserBubble.tsx`? The style in `useStyles()` is provided below
```
typingLoader: {
    position: 'relative',
    height: '2rem',
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
    margin: 'auto',
    padding: '0 0.5rem',
    '& div': {
      display: 'inline-block',
      width: '0.6rem',
      height: '0.6rem',
      borderRadius: '50%',
      background: '#ddd',
      margin: '0 0.3rem',
      animation: 'typing-loader-animation 1.2s cubic-bezier(0, 0.5, 0.5, 1) infinite',
      position: 'relative',
      '&:nth-child(1)': {
        animationDelay: '-0.24s',
      },
      '&:nth-child(2)': {
        animationDelay: '-0.12s',
      },
      '&:nth-child(3)': {
        animationDelay: '-0',
      },
    },
  },
  '@keyframes typing-loader-animation': {
    '0%': {
      top: '0rem',
    },
    '25%': {
      top: '0.5rem',
    },
    '100%': {
      top: '-0.5rem',
    },
  },
```
```
<div className={classes.typingLoader}>
  <div></div>
  <div></div>
  <div></div>
</div>
```

Reply:
Thanks I was just being lazy as this css was from my another project.